provision/acc_provision/acc_provision.py

```diff
@@ -1171,7 +1283,11 @@ def generate_kube_yaml(config, operator_output, operator_tar, operator_cr_output
 
 
 def generate_apic_config(flavor_opts, config, prov_apic, apic_file):
+    apic=None
+    if prov_apic is not None:
+        apic = get_apic(config)
     configurator = ApicKubeConfig(config)
+    ApicKubeConfig.init_apic(config, apic)
```

Comment:
We are calling "get_apic()" in L1306 (and you are already initializing apic to None in the ApicKubeConfig), so instead of doing all of the above we just need to call:

    ApicKubeConfig.init_apic(config, apic)

after L1306.